dbt_platform_helper/commands/environment.py

```diff
@@ -326,30 +332,77 @@ def get_cert_arn(session, env_name):
 
 
 @environment.command()
-@click.option("--vpc-name")
-@click.option("--name", "-n", multiple=True, required=True)
+@click.option("--vpc-name", hidden=True)
+@click.option("--name", "-n", required=True)
 def generate(name, vpc_name):
-    ensure_cwd_is_repo_root()
+    if vpc_name:
+        click.secho(
+            f"This option is deprecated. Please add the VPC name for your envs to {PLATFORM_CONFIG_FILE}",
+            fg="red",
+        )
+        raise click.Abort
```

Comment:
Ohh, not for this pull request, but maybe we could replace all the `exit 1`s with this.